examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/index.html

```diff
@@ -30,12 +30,40 @@ <h2>
 						Checkbox
 					</a>
 				</li>
+				<li>
+					<a href="#checkboxPopup">
+						Checkbox Small
```

Comment:
How about calling it 
Small - Two button?

There should be one more Small checkbox example in future - one button.

Reply:
Ok